frontend/src/screenControllers/PlayerScreenController/PlayerScreenController.js

```diff
@@ -42,6 +42,14 @@ export default function PlayerScreenController() {
         />
       );
 
+    case 'connection-refused-name-taken':
+      return (
+        <PlayerErrorScreen
+          bigText="That name is already taken"
+          smallText="Try and be creative next time"
+        />
+      );
+
```

Comment:
In the future maybe we could make this message appear in a modal so the user can just tap out of it and adjust their name without needing to click Join and enter the code again

Reply:
I'm looking into this as I really like it as an idea. I think I could add a modal to the player join screen. But (I think) it would require removing the automatic players screen change for `pending-connection` on submit to do it which wouldn't be a *great* experience for people with slow connections.

That being said I think I've got figured out how to do this and will be looking at it more tomorrow.

Reply:
@I3uckwheat what are your thoughts on this? Do you think keeping the join code inputted is a better feature than having the player's screen change as soon as they hit submit?

Reply:
This is definitely not in scope for this, but something that should be done at a later time.

Reply:
This is definitely not in scope for this, but something that should be done at a later time.